scripts/cli/cmd/centrifuge.js

```diff
@@ -0,0 +1,51 @@
+const ethers = require('ethers');
+
+const constants = require('../constants');
+const CentrifugeHandlerContract = require("../../../build/contracts/CentrifugeAssetHandler.json");
+const BridgeContract = require("../../../build/contracts/Bridge.json");
+
+async function getHash(cli) {
+    const centHandler = new ethers.Contract(cli.centAddress, CentrifugeHandlerContract.abi, cli.mainWallet);
+    
+    try {
+        const res = await centHandler.getHash(ethers.utils.hexZeroPad(cli.hash, 32));
+        console.log(`The hash ${cli.hash} was ${res ? "found!" : "NOT found!"}`);
+    } catch (e) {
+        console.log({ e });
```

Comment:
Please call `process.exit(1)`, same below. It ensures future tests fail correctly.